e2e/pages/session-search.po.ts

```diff
@@ -19,14 +20,17 @@ export class SessionSearchPage {
     private listingRequestsTablePaginator = new Paginator(element(by.id('hearings-part-table-paginator')));
     private listHearingButton = element(by.id('listHearingButton'));
     public assignButton = element(by.id('assign'));
+    public startTime = element(by.id('startTime'));
+    private elementHelper = new ElementHelper();
 
     async clickAssignButton() {
         await browser.wait(ExpectedConditions.elementToBeClickable(this.assignButton), Wait.normal, 'Assign button not visible')
         await this.assignButton.click()
     }
 
-    async acceptAssignWithCurrentTime() {
+    async acceptAssignWithCurrentTime(time) {
         await browser.wait(ExpectedConditions.elementToBeClickable(this.listHearingButton), Wait.normal, 'List hearing button not visible');
+        await this.elementHelper.typeValue(this.startTime, time);
```

Comment:
Please use Logger class and log what time is being put into this element. However, if typeValue function already does the logging then it's fine.

Reply:
typeValue already does the logging :)